app/logic/landingPage.py

```diff
@@ -35,7 +35,7 @@ def getActiveEventTab(programID):
     program = Program.get_by_id(programID)
     if program.isBonnerScholars:
         return "bonnerScholarsEvents"
-    elif program.isStudentLed:
-        return "studentLedEvents"
+    elif program.isVolunteerOpportunities:
+        return "volunteerOpportunities"
     else:
-        return "otherEvents"
+        return "celtsLabor"
```

Comment:
Because we won't have `isVolunteerOpportunities`, you might need to refactor how the active event tab selection works. You might be able to simplify it by having it not be based on program in all cases, but an explicit setting on each landing page button.